atom/feature_engineering.py

```diff
@@ -529,6 +548,11 @@ class FeatureSelector(BaseEstimator, TransformerMixin, BaseTransformer, FSPlotte
             - "SFS"" Sequential Feature Selection.
             - "RFE": Recursive Feature Elimination.
             - "RFECV": RFE with cross-validated selection.
+            - "PSO" : Perform binary-particle swarm optimization for feature selection
```

Comment:
What does binary mean in these defintions?

Reply:
the standard form of these optimizers work on continuous ( sometimes bounded ) search spaces. Feature selection being a binary task, these algos are specially modified. The 'binary' keyword is usually used to differentiate between the two forms.